pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aead_composite_common.go

```diff
@@ -6,6 +6,9 @@ SPDX-License-Identifier: Apache-2.0
 
 package subtle
 
+// package subtle provides the core crypto primitives to be used by composite primitives. It is intended for internal
```

Comment:
@Baha-sk curious: why is it called `subtle`?

Reply:
Tink Primitives call their core package this way, it contains the core primitive logic intended to be used by the facing API package. It's not internal to allow interop between different Primitives (and, I think, expose calls to the core primitives by advanced users). Tink doesn't guarantee subtle packages won't change in the future and they're not part of the official API. They're not intended for external use. @llorllale